docs/web/docs/1-getting_started/7-github.mdx

```diff
@@ -316,25 +318,31 @@ Typical pull-requests review flow requires reviewers approval. Action such as au
 
 #### Open a pull request on your iambic-template repo
 
-1. `git checkout -b task/change_description origin/main`
-2. Edit one of your description within your templates, like `resources/github/iam_role_lambda.yaml`. <!-- #TODO edit to what?? -->
-3. Add your changes to your branch, and push it to github:
+1. `default_branch=$(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')`
+2. `git checkout -b task/change_description "$default_branch"`
+3. Edit one of your description within your templates, like `resources/github/iam_role_lambda.yaml`. <!-- #TODO edit to what?? -->
+4. Add your changes to your branch, and push it to github:
 	```bash
 	git add resources/github/iam_role_lambda.yaml
 	git commit -m "Change Description"
 	git push origin HEAD
 	```
-4. Create a pull request.
-5. The installed GitHub App will comment with plan the changes. Any time you want iambic github action to re-plan the changes, you can comment `iambic plan`.
-6. The plan will be a comment in line, as long as it is less than 65kb. (Be very careful with big plans because those are difficult for reviewers to review.) <!-- #TODO So then what? What's the alternative? -->
-7. Add reviewers to review your PR and plan that is in the PR comments.
-8. Once your reviewer approves your changes, add comment `iambic apply` to have iambic apply the changes. If the changes can be successfully applied, the PR will be automatically merged into main as well.
+5. Create a pull request.
+6. The installed GitHub App will comment with plan the changes. Any time you want iambic github action to re-plan the changes, you can comment `iambic plan`.
+7. The plan will be a comment in line, as long as it is less than 65kb. (Be very careful with big plans because those are difficult for reviewers to review.) <!-- #TODO So then what? What's the alternative? -->
```

Comment:
Not holding anything up but I thought if the output of plan exceeded the make size for a comment we wrote it to a gist? Not sure if that's actually implemented or has even been formally defined though.